src/components/form/HdGoogleAutocomplete.vue

```diff
@@ -249,7 +253,15 @@ export default {
       if (this.required && this.isEmpty) {
         this.showError(this.t.FORM.VALIDATION.REQUIRED);
       } else {
-        this.hideError();
+        const [firstFailingValidation] = this.customValidators
```

Comment:
Good point, I will adapt it. 

If we ever want to support multiple validation error messages for an input field, we can just change it back later.

Reply:
This will not work as suggested, though, since `firstFailingValidation` will end up being the _function_ and not the result.

This made me realize a problem in the abstraction – to get the error message, you need to invoke the function. Instead I suggest to change it to `customRules` which look like this:

```
[
  { 
    validate: value => value.length >= 10, 
    errorMessage: atLeastTenCharsErrorMessage 
  },
  // ...more rules
]
```
and then:
```
const firstFailingRule = this.customRules.find(({ validate }) => !validate(this.value));
...
...
this.showError(firstFailingRule.errorMessage);
```

See my latest changes for this new implementation. @ilyasmez thoughts? :)

Reply:
I had the same feeling about the function, nice that you fixed that as well! :D